### DISCUSSION TITLE
Multi Domain Tracking & anonymize_ip: true

### DISCUSSION THREAD

**Discussion by vivekshotti** [MatteoGabriele/vue-gtag#519] [Q&A]
Is the ```anonymize_ip: true``` a necessary parameter to track multiple domains?
I was wondering if we could simply do this to track two domains:

```(javascript)
app.use(
  VueGtag,
  {
    includes: [{ id: "G-id1" }],
    config: { id: "G-id2" },
  },
  router
);
```

**Comment by MatteoGabriele** [comment]
I am not sure if this field is necessary for Google, but regarding this library, it is perfectly acceptable.
